// @google-cloud/game-servers/build/src/index.js
"use strict";
// Copyright 2020 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//
// ** This file is automatically generated by synthtool. **
// ** https://github.com/googleapis/synthtool **
// ** All changes to this file may be overwritten. **
Object.defineProperty(exports, "__esModule", { value: true });
exports.protos = exports.RealmsServiceClient = exports.GameServerDeploymentsServiceClient = exports.GameServerConfigsServiceClient = exports.GameServerClustersServiceClient = exports.v1beta = exports.v1 = void 0;
const v1 = require("./v1");
exports.v1 = v1;
const v1beta = require("./v1beta");
exports.v1beta = v1beta;
const GameServerClustersServiceClient = v1.GameServerClustersServiceClient;
exports.GameServerClustersServiceClient = GameServerClustersServiceClient;
const GameServerConfigsServiceClient = v1.GameServerConfigsServiceClient;
exports.GameServerConfigsServiceClient = GameServerConfigsServiceClient;
const GameServerDeploymentsServiceClient = v1.GameServerDeploymentsServiceClient;
exports.GameServerDeploymentsServiceClient = GameServerDeploymentsServiceClient;
const RealmsServiceClient = v1.RealmsServiceClient;
exports.RealmsServiceClient = RealmsServiceClient;
exports.default = {
    v1,
    v1beta,
    GameServerClustersServiceClient,
    GameServerConfigsServiceClient,
    GameServerDeploymentsServiceClient,
    RealmsServiceClient,
};
const protos = require("../protos/protos");
exports.protos = protos;
